tests/util/iostreamContainerUTest.cxxtest

```diff
@@ -31,28 +31,28 @@ using std::string;
 class printContainerUTest : public CxxTest::TestSuite
 {
```

Comment:
The changes in this file fix a bug related to the _N symbol being already used under cygwin.  Simply renamed N and _N it to NN and _NN respectively.